doc/contributor/howto-guide-generated-code-maintenance.md

```diff
@@ -66,6 +66,16 @@ go -C generator/ run ./cmd/sidekick refreshall -project-root .. && cargo fmt
 
 Then run the unit tests and send a PR with whatever changed.
 
+## Update golden files
+
+The golden files live under `generator/testdata`.
+
+Run:
+
+```bash
+go -C generator test ./...
```

Comment:
Maybe something about how to diff the files and commit the changes when you are happy with the differences?

Reply:
Good point, added a line to run `git diff generator/testdata` to verify the changes.